data/presets/presets/barrier/spike.json

```diff
@@ -0,0 +1,12 @@
+{
+    "icon": "maki-fence",
+    
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "barrier": "spike"
+    },
+    "name": "Spike",
```

Comment:
These things go by many names, but I’m unaware of it being called simply “spike”. The most common term seems to be “spike strip”.

Reply:
Thank you, I just renamed it to "spike strip". Any other suggestions.

Reply:
ok, so here is the documentation:
https://wiki.openstreetmap.org/wiki/Tag:barrier=spikes

So: 
The tags should be `"barrier": "spikes"`.  
The filename should be `spikes.json`.
The name should be `"Spike Strip"`.  
The geometry should be `["vertex"]`.

Reply:
Made these changes in the file.

Reply:
> Made those changes in the file.

Sorry but the file is still not named correctly or using the correct tag.  I'm going to close this PR.